package.json

```diff
@@ -85,72 +84,63 @@
     "@alfresco/adf-process-services-cloud": "3.6.0",
     "@alfresco/adf-testing": "3.6.0",
     "@alfresco/js-api": "3.6.0",
-    "@angular/animations": "^7.2.15",
-    "@angular/cdk": "7.3.7",
-    "@angular/common": "^7.2.15",
-    "@angular/compiler": "^7.2.15",
-    "@angular/core": "^7.2.15",
-    "@angular/flex-layout": "^7.0.0-beta.24",
-    "@angular/forms": "^7.2.15",
-    "@angular/http": "^7.2.15",
-    "@angular/material": "^7.3.7",
-    "@angular/material-moment-adapter": "^7.3.7",
-    "@angular/platform-browser": "^7.2.15",
-    "@angular/platform-browser-dynamic": "^7.2.15",
-    "@angular/router": "^7.2.15",
+    "@angular/animations": "^8.2.14",
+    "@angular/cdk": "8.2.3",
+    "@angular/common": "^8.2.14",
+    "@angular/compiler": "^8.2.14",
+    "@angular/core": "^8.2.14",
+    "@angular/flex-layout": "^8.0.0-beta.27",
+    "@angular/forms": "^8.2.14",
+    "@angular/material": "^8.2.3",
+    "@angular/material-moment-adapter": "^8.2.3",
+    "@angular/platform-browser": "^8.2.14",
+    "@angular/platform-browser-dynamic": "^8.2.14",
+    "@angular/router": "^8.2.14",
     "@mat-datetimepicker/core": "^2.0.1",
     "@mat-datetimepicker/moment": "^2.0.1",
     "@ngx-translate/core": "^11.0.0",
-    "@types/request": "^2.48.3",
     "adf-monaco-extension": "0.0.8",
     "adf-tslint-rules": "0.0.6",
     "chart.js": "2.5.0",
-    "classlist.js": "1.1.20150312",
-    "core-js": "^2.5.4",
-    "custom-event-polyfill": "0.3.0",
-    "hammerjs": "2.0.8",
+    "hammerjs": "^2.0.8",
```

Comment:
Do we need this hammerjs at the end? I saw it was removed from everywhere except this one.

Reply:
We need it only on the application level. We have no api or components that are actually depending on the hammer in ADF

Reply:
👍